app.js

```diff
@@ -125,13 +125,15 @@ app.use(function(err, req, res, next) {
         .json({ message: err.data[0].message, path: err.data[0].path });
     } else {
       // unexpected error, so log it
-      logger.error({ url: req.originalUrl, message: err.message });
+      logger.debug(err);
```

Comment:
Why the change in severity level for logging?

Reply:
Thanks for catching this! I meant to add additional logging at the `debug` level (which will only be done in non-prod environments, as configured in `utils/logger.js`) and have reverted it so that it still writes to `logger.error` in all environments.